addons/website_quote/views/report_saleorder.xml

```diff
@@ -8,6 +8,35 @@
             <p t-field="doc.validity_date"/>
         </div>
     </xpath>
+    <xpath expr="//div[@class='page']/p[@t-if='doc.fiscal_position and doc.fiscal_position.note']" position="after">
```

Comment:
for node selection using xpath expr don't rely on @class @href like attributes which have higher probability to change so there is always potential risk to break expr. It would be better to use @id or @name attribute.
also don't use non standard html attributes like @t-if